source/getting-started-edge/index.rst

```diff
@@ -89,6 +89,142 @@ resource usage. The dashboard looks  something like this:
+    |              |     "updated_at": null,                                               |
+    |              |     "id": "e8892924-649a-4beb-aa46-9e16f6331dab",                     |
+    |              |     "lease_id": "8aad6912-2eb5-4140-812f-123e5cb56ca3",               |
+    |              |     "event_type": "start_lease",                                      |
+    |              |     "time": "2021-06-24T15:45:00.000000",                             |
+    |              |     "status": "UNDONE"                                                |
+    |              | }                                                                     |
+    | id           | 8aad6912-2eb5-4140-812f-123e5cb56ca3                                  |
+    | name         | my-first-lease                                                        |
+    | project_id   | a5f0758da4a5404bbfcef0a64206614c                                      |
+    | reservations | {                                                                     |
+    |              |     "created_at": "2021-06-24 15:43:36",                              |
+    |              |     "updated_at": "2021-06-24 15:43:36",                              |
+    |              |     "id": "500e0c36-2089-46a5-bf7c-cc46e5f65a0d",                     |
+    |              |     "lease_id": "8aad6912-2eb5-4140-812f-123e5cb56ca3",               |
+    |              |     "resource_id": "48001fa1-ccb5-4e30-b511-a90455930776",            |
+    |              |     "resource_type": "device",                                        |
+    |              |     "status": "pending",                                              |
+    |              |     "missing_resources": false,                                       |
+    |              |     "resources_changed": false,                                       |
+    |              |     "resource_properties": "[\"==\", \"$vendor\", \"Raspberry Pi\"]", |
+    |              |     "before_end": "default",                                          |
+    |              |     "min": 1,                                                         |
+    |              |     "max": 1                                                          |
+    |              | }                                                                     |
+    | start_date   | 2021-06-24T15:45:00.000000                                            |
+    | status       | PENDING                                                               |
+    | trust_id     | ec2a893aa0494d72bcc5fbb3b73e7e66                                      |
+    | updated_at   | 2021-06-24 15:43:36                                                   |
+    | user_id      | b8f54aa95b96b9fb69e31a3e39df6a7bad29581439cf8bd8c9d59d9d7d048f3a      |
+    +--------------+-----------------------------------------------------------------------+
+
+Look for the *reservations* entry, and within this item find the *id* entry. In
```

Comment:
Maybe a quick note that reservations can take up to a minute to start. Once the lease is in ACTIVE state, that means you can use it.